site/docs/guides/user-input.md

```diff
@@ -40,10 +40,10 @@ if gamepad & BUTTON_RIGHT != 0 {
 ```
 
 ```go
-var gamepad = *GAMEPAD1;
+var gamepad = *w4.GAMEPAD1
 
-if (gamepad & BUTTON_RIGHT != 0) {
-    trace("Right button is down!");
+if gamepad&w4.BUTTON_RIGHT != 0 {
```

Comment:
```suggestion
if gamepad & w4.BUTTON_RIGHT != 0 {
```

Reply:
`gofmt` removes the spaces around the `&` on it's own. No realy use adding them back in. It's the same with the suggestion below.